tools/Performance/DynamoPerformanceTests/GraphNameColumn.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+
+namespace DynamoPerformanceTests
+{
+    public class GraphNameColumn : IColumn
+    {
+        public GraphNameColumn() { }
+        
+        public string ColumnName => "Graph";
+
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase)
+        {
+            var path = benchmarkCase.Parameters["DynamoFilePath"].ToString();
+            var fi = new FileInfo(path);
+            return Path.GetFileNameWithoutExtension(fi.Name);
+        }
+
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase, SummaryStyle style) => GetValue(summary, benchmarkCase);
+        
+        public string Id => nameof(GraphNameColumn);
```

Comment:
I'm okay if we don't add new doc strings for these members that come from benchmarkdotnet.